packages/components/src/components/link-button/component.tsx

```diff
@@ -97,11 +102,6 @@ export class KolLinkButton implements Props {
 	 */
 	@Prop() public _label!: LabelWithExpertSlotPropType;
 
-	/**
-	 * Listen on an aria-current event with this value. If the value matches the current value and the href is the same as the current url, the aria-current attribute will be set to current value.
-	 */
-	@Prop() public _listenAriaCurrent?: AriaCurrentPropType;
```

Comment:
@sdvg Hi Stefan, ich finde diesen Prop Name intiutiver.,

Reply:
@deleonio Hallo Martin, könntest du das bitte noch etwas näher beschreiben? Diese Property haben (brauchen) wir ja in der Form nicht mehr?

Reply:
Für mich ist hier das Lauschen auf den Event/Änderung klarer. Alles gut.